.github/workflows/test.yml

```diff
@@ -9,8 +9,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: jtdor/build-deb-action@v1.0.0
-        with:
-          docker-image: debian:unstable-slim
-          buildpackage-opts: --build=binary --no-sign
+      # FIXME massive hack, just trying to see if this works
+      - name: build and test
+        run: |
+          set -ex
+          git fetch --all
+          git branch pristine-tar origin/pristine-tar
+          git branch upstream/latest origin/upstream/latest
+          CONTAINER_ID="$(docker run --rm --privileged --detach -e DEBIAN_FRONTEND=noninteractive -v "$GITHUB_WORKSPACE:/github/workspace" --workdir=/github/workspace debian:stable-slim tail -f /dev/null)"
+          DEBARCH="$(docker exec "$CONTAINER_ID" dpkg --print-architecture)"
+          docker exec "$CONTAINER_ID" apt-get update
+          docker exec "$CONTAINER_ID" apt-get install -y git-buildpackage build-essential git autopkgtest
```

Comment:
`git-buildpackage` already directly depends on `git`, which you can verify with `apt-cache show git-buildpackage`, although it won't hurt anything to list it here.  I checked because I was expecting that `build-essential` would also already be installed and the line could be much shorter, but `git-buildpackage` depends on `devscripts`, but `devscripts` only _recommends_ `build-essential`.

In other words, ignore this - I am merely quipping aloud.

Reply:
> `git-buildpackage` already directly depends on `git`

Bah, how was I supposed to know that! 😉 Thanks. I actually think most of these comments are because I started out with sbuild and walked it back to gbp.